recipe/meta.yaml

```diff
@@ -21,17 +21,20 @@ source:
 build:
   number: 2
   run_exports:
-    - {{ pin_subpackage('pytorch', max_pin='x.x.x') }}
+    - {{ pin_subpackage('pytorch_openmpi', max_pin='x.x.x') }}
   skip: true  # [win]
 
 outputs:
-  - name: pytorch
+  - name: pytorch_openmpi
```

Comment:
I wasn't planning to rename the package. I don't think things will work well if we do because then it's not clear how to fulfill the dependencies of other packages that require pytorch.

Let's leave the name as `pytorch` unless you have a different specific plan.